### PR TITLE
Add spaces before the label

### LINKED ISSUE #15098: Spaces issue in radio button
### Describe the bug
I have noticed that there's padding and spaces issue in radio buttons.

### To Reproduce

1. Go to Settings
2. Then on manage your settings
3. You'll see the problem

### Expected behavior

I was expecting the same consistent padding and spaces for all the buttons.

### Screenshots

Before:

image

After:

image

### Server configuration

- Operating system: Windows
- Web server: XAMPP
- PHP version: 7.3.0
- phpMyAdmin version: 5.0.0-dev

### Client configuration

- Browser: Firefox, Google Chrome
- Operating system: Windows

### PR DESCRIPTION
**Pull request by AghaSaad04**
Signed-off-by: Agha Saad Fraz <agha.saad04@gmail.com>

### Description

I have added spaces before the labels of the radio buttons to make them consistent throughout the theme.

Fixes #15098 

Before submitting pull request, please review the following checklist:

- [ ] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [ ] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [ ] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [ ] Every commit has a descriptive commit message.
- [ ] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ ] Any new functionality is covered by tests.

### CODE REVIEW

**Comment by ibennetch** [comment]
I think this is not an ideal solution, because adding this to the translated string doesn't make much sense for translators and depends on each translation maintaining the markup. I think a better solution would be putting the space outside of the translated (`__()`) area.

Plus, I don't see a need for a nonbreaking space here, wouldn't a regular ` ` character work just as well?

**Comment by AghaSaad04** [comment]
Sir @ibennetch, I tried both a regular space and space outside the translated area but those were not working on my side. So, I went for a non-breaking space.

File: prefs_manage.php [hunk 1]
```diff
@@ -239,15 +239,15 @@
     , '<input type="hidden" name="json" value="">'
     , '<input type="radio" id="import_text_file" name="import_type"'
     , ' value="text_file" checked="checked">'
-    , '<label for="import_text_file">' . __('Import from file') . '</label>'
+    , '<label for="import_text_file">' . __('&nbsp;Import from file') . '</label>'
```
Comment (mauriciofauth) [inline prefs_manage.php]: It's not a good idea to add this to a translatable string. Adding a space between the input and label tags should probably fix this.

Comment (AghaSaad04) [reply prefs_manage.php]: Okay, Sir @mauriciofauth. I have removed that non-breaking space and added space between the input and label.

**Comment by mauriciofauth** [comment]
Hi @AghaSaad04. Thank you for your contribution.

The pull request #15055 fixed the issue by extracting the HTML to a Twig template. So that's why I closed this PR.

**Pull request closed without merging**